swarm/initialize.py

```diff
@@ -94,9 +118,5 @@ def get_models_from_hive(hive_uri):
 
         return data["language_models"] + data["models"]
     except Exception as e:
-        print(e)
-        logging.error(e)
+        print(f"Failed to fetch known model list from {hive_uri}: {e}")
         return []
-
-
-asyncio.run(init())
```

Comment:
without this how does `init` get invoked? (This file runs as a module `python-m swarm.intialize`) Perhaps we add a `if __name__ == "__main__":` for correctness sake but this code isn't (or shouldn't be imported anywhere.

Reply:
Error when reviewing code.